tests/helpers/test_device_registry.py

```diff
@@ -26,19 +26,41 @@ def registry(hass):
 async def test_get_or_create_returns_same_entry(registry):
     """Make sure we do not duplicate entries."""
     entry = registry.async_get_or_create(
-        [['bridgeid', '0123']], 'manufacturer', 'model',
-        [['ethernet', '12:34:56:78:90:AB:CD:EF']])
+        {('bridgeid', '0123')}, 'manufacturer', 'model',
```

Comment:
for the sake of giving good examples, we should not use `bridgeid` here. All identifiers need to be scoped and not be generic names. If Hue and Deconz use same bridge ids, it shouldn't match devices.

Reply:
That is part of the deconz PR. I don't want to do same change in two different PRs

Reply:
ok